### PR TITLE
[13.0][MIG] maintenance_equipment_always_fold

### PR DESCRIPTION
**Pull request by ChristianSantamaria**
Standard migration to v13

### CODE REVIEW

**Comment by dalonsod** [comment]
@OCA/maintenance-maintainers in this PR and others (v13, v14) runbot is missing. Could you check? Thanks!

File: maintenance_equipment_always_fold/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "summary": "Equipment categories will always be folded if I indicate it "
+    "with an always_fold Boolean; if not, let him behave as before",
+    "author": "Odoo Community Association (OCA), Solvos",
+    "maintainers": ["dalonsod"],
+    "license": "AGPL-3",
+    "version": "13.0.1.0.0",
+    "category": "Equipments",
+    "website": "https://github.com/OCA/maintenance",
+    "depends": ["maintenance"],
+    "data": ["views/maintenance_views.xml", "demo/maintenance_demo.xml"],
```
Comment (etobella) [inline maintenance_equipment_always_fold/__manifest__.py]: Demo data should go to demo

Comment (ChristianSantamaria) [reply maintenance_equipment_always_fold/__manifest__.py]: Yes, my fault, thanks for the review

**Review by etobella — APPROVED** [review]

File: maintenance_equipment_always_fold/__manifest__.py (hunk 1 above)
Comment (etobella) [reply maintenance_equipment_always_fold/__manifest__.py]: Well, it came from history, so my fault as I didn't found it in the first review :wink:

**Comment by ChristianSantamaria** [comment]
Hi @mariadforgeflow, could you take a look at it?

**Review by mariadforgeflow — APPROVED** [review]
LGTM

**Pull request merged**